src/molecules/Hero/hero.module.css

```diff
@@ -0,0 +1,16 @@
+.hero {
```

Comment:
Please arrange properties in alphabetical order.

Reply:
Done.